test/CLI.js

```diff
@@ -810,6 +810,17 @@ module.exports = (testRunner, jsonFixtures, csvFixtures) => {
     });
   });
 
+  testRunner.add('should support flattening JSON with nested arrays using the flatten transform, when arrays a longer in non-0 index entries', (t) => {
+    const opts = '--flatten-objects --flatten-arrays';
+
+    exec(`${cli} -i "${getFixturePath('/json/flattenArraysReversed.json')}" ${opts}`, (err, stdout, stderr) => {
```

Comment:
Why are the fixture and the csv called `flattenArraysReversed`? Are we reversing anything here?
`flattenArraysOfDifferentLengths` or something like that might describe the intent better.

Reply:
No reason other than it was like the other test but the rows were reversed, wasn't fixated on a name.